cpython/download_pypi_top.py

```diff
@@ -67,7 +67,7 @@ def download_sdist(dst_dir, index, proj, nproject):
 def main():
     start_time = time.monotonic()
 
-    if len(sys.argv) < 2:
+    if len(sys.argv) < 2 or sys.argv[1] in ['-h', '--help']:
         print("Usage: %s directory [count]" % sys.argv[0])
         sys.exit(1)
```

Comment:
I write these 3 lines what I'm too lazy to write proper code. If you want a real CLI, please use the argparse module instead.

Reply:
sure thing, updated